tests/utils.py

```diff
@@ -27,17 +34,46 @@ def login(flask_client) -> User:
     return user
 
 
-def create_user(flask_client) -> User:
-    # create user, user is activated
+def random_token(length: int = 10) -> str:
+    return "".join(random.choices(string.ascii_lowercase + string.digits, k=length))
+
+
+def create_random_user() -> User:
+    email = "{}@{}.com".format(random_token(), random_token())
     return User.create(
-        email="a@b.c",
+        email=email,
         password="password",
         name="Test User",
         activated=True,
         commit=True,
     )
 
 
+def create_random_alias(user: User) -> Alias:
```

Comment:
I think we have a similar method called `Alias.create_new_random`

Reply:
Ok. I'll use it.